deploy/templates/nstemplatetiers/appstudio/ns_tenant.yaml

```
@@ -269,6 +269,26 @@ objects:
     podSelector: {}
     policyTypes:
     - Ingress
+# ServiceAccount and RoleBindings for running Pipelines.
+# appstudio-pipelines-runner-clusterrole is deployed by the pipeline-service component.
+- apiVersion: v1
+  kind: ServiceAccount
+  metadata:
+    name: appstudio-pipeline
+    namespace: ${USERNAME}-tenant
+- apiVersion: rbac.authorization.k8s.io/v1
+  kind: RoleBinding
+  metadata:
+    name: appstudio-pipelines-runner-rolebinding
+    namespace: ${USERNAME}-tenant
+  roleRef:
+    apiGroup: rbac.authorization.k8s.io
+    kind: ClusterRole
+    name: appstudio-pipelines-runner-clusterrole
```

https://github.com/codeready-toolchain/member-operator/blob/abc7052dba35ef660902664fd843c38a6ec37882/config/appstudio-pipelines-runner/base/appstudio_pipelines_runner_role.yaml#L2-L5
```suggestion
    name: appstudio-pipelines-runner
```